community/minutes/2025-02-05.md

```diff
@@ -0,0 +1,64 @@
+---
+tags: [meeting-notes]
+title: '2025-02-05'
+---
+# conda-forge core meeting 2025-02-05
+
+Add new agenda items under the `Your __new__() agenda items` heading
+
+- [Zoom link](https://zoom.us/j/9138593505?pwd=SWh3dE1IK05LV01Qa0FJZ1ZpMzJLZz09)
+- [What time is the meeting in my time zone](https://dateful.com/convert/utc?t=5pm)
+- [Previous meetings](https://conda-forge.org/community/minutes/)
+
+## Attendees
+
+| Name                    | Initials | GitHub ID        | Affiliation                 |
+| ----------------------- | -------- | ---------------  | --------------------------- |
+| Wolf Vollprecht         | WV       | wolfv            | prefix.dev                  |
+| Dasha Gurova            | DG       | dashagurova      | Anaconda                    |
+| John Kirkham            | JK       | jakirkham        | NVIDIA/cf                   |
+| Klaus Zimmermann        | KZ       | zklaus           | Quansight                   |
+| Jannis Leidel           | JL       | jezdez           | Anaconda/cf                 |
+|                         |          |                  |                             |
+|                         |          |                  |                             |
+
+X people total
+
+### Standing items
+
+- [ ]
+
+### From previous meeting(s)
+
+- [ ]
+
+### Active votes
+
+- [ ]
+
+### Your __new__() agenda items
+
+- [x] (JK) CUDA 12.8 migration
+  - https://github.com/conda-forge/conda-forge-pinning-feedstock/issues/6980
+- [x] (WV) sigstore is deployed on prefix.dev
+    - Talked with OpenSSF today
+- [x] (WV) CEP open for huge repodata overhaul
+    - https://github.com/conda/ceps/pull/111
+- [x] (WV) rattler-build update (types, lists, ..)
+    - https://github.com/conda/ceps/pull/110 
+- [x] (JK) Conda + Python 3.13
+- [x] Discussion about an in-person sprint or summit
+    - Would like to host a gathering of Conda and conda-forge
+    - Could push through CEPs in a more concerted effort
+    - Integrate newer folks into the community
+    - Venue? Maybe SciPy 2025? Or a separate venue unto itself
```

Comment:
Niiice, this is exciting! Who proposed this?

Reply:
There wasn't a specific proposer. It came up organically in discussion

Would be good to discuss this more and formulate a plan. Next step is to start a doc